website/static/api/mesh_object_declarative_import.html

```diff
@@ -576,7 +576,7 @@ <h4 id="mesh_object_declarative_import">meshObject Declarative Import</h4>
 Vary: Access-Control-Request-Headers
 Content-Type: application/vnd.meshcloud.api.meshobjects.v1+json
 X-Content-Type-Options: nosniff
-X-XSS-Protection: 1; mode=block
+X-XSS-Protection: 0
```

Comment:
@sttomm can you review that whether this is related to library upgrades

Reply:
yes, it is related to upgrading to Spring Boot 3.0 (and especialyl Spring Security 6). Setting this header to 0 is the new recommendation by OWASP. So I think this change is a good one. It seems like the old default did more harm than good in the end. So it should be good, that it is set to 0 now. https://github.com/spring-projects/spring-security/issues/9631